Bindings/Java/Matlab/Hopper_Device/BuildDevice.m

```diff
@@ -0,0 +1,188 @@
+function device = BuildDevice(varargin)
+% Build an OpenSim model of an assistive device.
+%
+% Optional parameters
+% -------------------
+% deviceType: 'active' for a PathActuator, 'passive' for a path spring.
+% isPropMyo: Control the active device using a PropMyoController
+%   (proportional myoelectric). If false, a PrescribedController is used
+%   instead. This parameter only applies if deviceType'active'.
+% control: For deviceType=='active' and isPropMyo==false, this variable
+%   specifies a prescribed control signal to use for the device. This should be
+%   a matrix where the first row contains time and the second row contains the
+%   control signal.
+% springStiffness: For deviceType=='passive', this is the stiffness of the
+%   spring in the passive device (in units of N/m).
+% passivePatellaWrap:  TODO remove, it is not used.
```

Comment:
I know this says TODO, but it looks like to me that only this line and L60 need to be removed to address this? If so, I'd lean towards taking it out now from this function.